sample-apps/python/django_frontend_service/frontend_service_app/views.py

```diff
@@ -2,6 +2,7 @@
 ## SPDX-License-Identifier: Apache-2.0
 import logging
 import os
+import base64
```

Comment:
Do we need to add this to `requirements.txt` and `ec2-requirements.txt`?

Reply:
@majanjua-amzn It's a built-in library from Python: https://docs.python.org/3/library/base64.html

The test run also confirmed it can import this library without add it as a dependency.